app/assets/stylesheets/colors.scss

```diff
@@ -0,0 +1,5 @@
+$dark-green: #335650; // dark-green
+$light-green: #9EAE8A; // light-green
+$light-brown: #CDBB93 ; //light-brown
```

Comment:
Color `#CDBB93` should be written as `#cdbb93`<br>Declaration should be terminated by a semicolon